README.md

```diff
@@ -131,6 +131,7 @@ rgbd_obstacle_layer:
     inf_is_valid: false          #default false, for laser scans
     clear_after_reading: true    #default false, clear the buffer after the layer gets readings from it
     voxel_filter: true           #default off, apply voxel filter to sensor, recommend on 
+    passthrough_filter: true     #default true, crop data by min & max height if voxel_filter off
```

Comment:
Need some docs about one versus the other and what takes priority if both are set to true (like say, in this file)